spec/spec_helper_acceptance.rb

```diff
@@ -144,10 +162,21 @@ def teardown_puppet_on(host)
   step "Purge puppet from #{host}"
   # Note pc1_repo is specific to the module's manifests. This is knowledge we need to clean
   # the machine after each run.
+  case host['platform']
+  when /debian|ubuntu/
+    on host, '/opt/puppetlabs/bin/puppet module install puppetlabs-apt', { :acceptable_exit_codes => [0,1] }
```

Comment:
Are there cases where we won't have the module installed already? Also minor nit, `on host, puppet('module', ...)`?